cmd/cel-shed/eds_store_stress.go

```diff
@@ -38,12 +45,15 @@ func init() {
 
 	pathFlagUsage := fmt.Sprintf("Directory path to use for stress test. Uses %s by default.", defaultPath)
 	edsStoreStress.Flags().String(edsStorePathFlag, path, pathFlagUsage)
+	edsStoreStress.Flags().String(pyroscopeEndpointFlag, "", "Pyroscope address")
```

Comment:
why not set default?

Reply:
Default is empty string, it will disable Pyroscope, in case no address is provided.